pylace/src/lib.rs

```diff
@@ -1179,6 +1179,71 @@ impl CoreEngine {
         Ok(())
     }
 
+    /// Remove Rows at the given indices.
+    ///
+    ///
+    /// Example
+    /// -------
+    ///
+    /// >>> import lace
+    /// >>> engine = lace.Engine('animals.rp')
+    /// >>> removed = engine.remove_rows(["wolf", "ox"])
+    /// >>> removed["index"]
```

Comment:
What is this line supposed to show in the example? Maybe a better one would be an assert showing that the new DF has two fewer rows?

Reply:
I added some code to make this test actually do something.